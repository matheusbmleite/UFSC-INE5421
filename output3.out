{
    "init_state": [
        "q0"
    ],
    "final_states": [
        [
            "q2",
            "q0",
            "q1",
            "q3"
        ],
        [
            "q0",
            "q1",
            "q3"
        ],
        [
            "q0",
            "q3"
        ]
    ],
    "type": "automaton",
    "states": [
        [
            "q0",
            "q1",
            "q3"
        ],
        [
            "q2",
            "q0",
            "q1"
        ],
        [
            "q0",
            "q1"
        ],
        [
            "q2",
            "q0",
            "q1",
            "q3"
        ],
        [
            "q0"
        ],
        [
            "q0",
            "q3"
        ]
    ],
    "alphabet": [
        "a",
        "b"
    ],
    "transitions": {
        "['q0', 'q1']": {
            "b": [
                "q0"
            ],
            "a": [
                "q2",
                "q0",
                "q1"
            ]
        },
        "['q2', 'q0', 'q1', 'q3']": {
            "b": [
                "q0",
                "q3"
            ],
            "a": [
                "q2",
                "q0",
                "q1",
                "q3"
            ]
        },
        "['q0']": {
            "b": [
                "q0"
            ],
            "a": [
                "q0",
                "q1"
            ]
        },
        "['q2', 'q0', 'q1']": {
            "b": [
                "q0",
                "q3"
            ],
            "a": [
                "q2",
                "q0",
                "q1"
            ]
        },
        "['q0', 'q1', 'q3']": {
            "b": [
                "q0",
                "q3"
            ],
            "a": [
                "q2",
                "q0",
                "q1",
                "q3"
            ]
        },
        "['q0', 'q3']": {
            "b": [
                "q0",
                "q3"
            ],
            "a": [
                "q0",
                "q1",
                "q3"
            ]
        }
    }
}