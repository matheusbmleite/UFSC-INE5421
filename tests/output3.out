{
    "alphabet": [
        "a",
        "b"
    ],
    "type": "automaton",
    "transitions": {
        "['q3', 'q0']": {
            "a": [
                "q3",
                "q0",
                "q1"
            ],
            "b": [
                "q3",
                "q0"
            ]
        },
        "['q0', 'q1']": {
            "a": [
                "q2",
                "q0",
                "q1"
            ],
            "b": [
                "q0"
            ]
        },
        "['q0']": {
            "a": [
                "q0",
                "q1"
            ],
            "b": [
                "q0"
            ]
        },
        "['q2', 'q3', 'q0', 'q1']": {
            "a": [
                "q2",
                "q3",
                "q0",
                "q1"
            ],
            "b": [
                "q3",
                "q0"
            ]
        },
        "['q2', 'q0', 'q1']": {
            "a": [
                "q2",
                "q0",
                "q1"
            ],
            "b": [
                "q3",
                "q0"
            ]
        },
        "['q3', 'q0', 'q1']": {
            "a": [
                "q2",
                "q3",
                "q0",
                "q1"
            ],
            "b": [
                "q3",
                "q0"
            ]
        }
    },
    "init_state": [
        "q0"
    ],
    "states": [
        [
            "q2",
            "q3",
            "q0",
            "q1"
        ],
        [
            "q0"
        ],
        [
            "q3",
            "q0",
            "q1"
        ],
        [
            "q3",
            "q0"
        ],
        [
            "q2",
            "q0",
            "q1"
        ],
        [
            "q0",
            "q1"
        ]
    ],
    "final_states": [
        [
            "q2",
            "q3",
            "q0",
            "q1"
        ],
        [
            "q3",
            "q0",
            "q1"
        ],
        [
            "q3",
            "q0"
        ]
    ]
}